packages/ui/package.json

```
@@ -5,5 +5,13 @@
   "license": "MIT",
   "devDependencies": {
     "config": "workspace:*"
+  },
+  "dependencies": {
+    "@babel/core": "^7.0.0",
```

```suggestion
  "peerDependencies": {
```

This is to make sure a single version of all these dependencies will be used in each app.